corehq/apps/integration/kyc/forms.py

```diff
@@ -0,0 +1,112 @@
+import json
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from crispy_forms import bootstrap as twbscrispy
+from crispy_forms import layout as crispy
+from crispy_forms.helper import FormHelper
+
+from corehq.apps.integration.kyc.models import (
+    KycConfig,
+    KycProviders,
+    UserDataStore,
+)
+from corehq.apps.userreports.ui.fields import JsonField
+from corehq.motech.models import ConnectionSettings
+
+
+class KycConfigureForm(forms.ModelForm):
+
+    class Meta:
+        model = KycConfig
+        fields = [
+            'user_data_store',
+            'other_case_type',
+            'provider',
+            'api_field_to_user_data_map',
+            'connection_settings',
+        ]
+
+    user_data_store = forms.ChoiceField(
+        label=_('User Data Store'),
+        required=True,
+        choices=UserDataStore.CHOICES,
+    )
+    other_case_type = forms.CharField(
+        label=_('Other Case Type'),
+        required=False,
+    )
+    provider = forms.ChoiceField(
+        label=_('Provider'),
+        required=True,
+        choices=KycProviders.choices,
+    )
+    api_field_to_user_data_map = JsonField(
+        label=_('API Field to User Data Map'),
+        required=True,
+        expected_type=list,
```

Comment:
The model field is `api_field_to_user_data_map = jsonfield.JSONField(default=dict)`. Should this be `dict`? i.e.
```suggestion
        expected_type=dict,
```

Reply:
The model field has been updated to take `list` as a default type [here](https://github.com/dimagi/commcare-hq/blob/ca8ff4ca5b46d454f2a9db688ccd7f8f71d6f486/corehq/apps/integration/kyc/models.py#L23). The expected format for the API mapping will be something like this:
```
[
  ...
  {
    "fieldName": "foo",
    "mapsTo": "bar",
    "source": "standard"
  }
  ...
]
```